core/includes/bootstrap.inc

```diff
@@ -3975,15 +3975,14 @@ function backdrop_get_complete_schema($rebuild = FALSE) {
     else {
       $schema = array();
       // Load the .install files to get hook_schema.
-      // On some databases this function may be called before bootstrap has
-      // been completed, so we force the functions we need to load just in case.
-      if (function_exists('module_load_all_includes')) {
-        // This function can be called very early in the bootstrap process, so
-        // we force the module_list() cache to be refreshed to ensure that it
-        // contains the complete list of modules before we go on to call
-        // module_load_all_includes().
-        module_list(TRUE);
-        module_load_all_includes('install');
+      // On some databases this function may be called before bootstrap has been
+      // completed, so we force the functions we need to load just in case. This
+      // function can be called very early in the bootstrap process, so we force
+      // the module_list() cache to be refreshed to ensure that it contains the
+      // complete list of modules.
+      $modules = module_list(TRUE);
+      foreach ($modules as $module) {
+        backdrop_include_file('module', $module, $module . '.install');
```

Comment:
The documentation comment for `backdrop_include_file()` says *Do not use this function to include an install file.* but that is what this code does.

Reply:
Thanks for pointing this out; new PR expands the comment to clarify where and when one can/not use the function to include an install file.